xbmc/cores/VideoPlayer/DVDCodecs/Overlay/contrib/cc_decoder708.cpp

```diff
@@ -197,23 +197,23 @@ void printTVtoBuf (cc708_service_decoder *decoder)
   if (empty)
     return; // Nothing to write
 
-  for (int i=0;i<75;i++)
+  for (auto& c : decoder->tv.chars)
```

Comment:
we iterate over rows

Reply:
still happening, `auto& c` is deduced to `unsigned char (c&)[I708_SCREENGRID_COLUMNS+1]`